.deepsource.toml

```diff
@@ -0,0 +1,8 @@
+version = 1
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
⛔ means that GitHub expects text files to end in one, and only one carriage return (\n).